src/os/windows/mod.rs

```diff
@@ -234,12 +234,12 @@ impl Library {
         }).map_err(|e| e.unwrap_or(crate::Error::GetProcAddressUnknown))
     }
 
-    /// Get a pointer to function or static variable by ordinal number.
+    /// Get a pointer to a function or static variable by ordinal number.
     ///
     /// # Safety
     ///
     /// Users of this API must specify the correct type of the function or variable loaded. Using a
-    /// `Symbol` with a wrong type is undefined.
+    /// `Symbol` with a wrong type causes undefined behaviour.
```

Comment:
Using a `Symbol` with a wrong type here cannot cause undefined behaviour in the way "causes undefined behaviour" is understood in the context of the language specification and frequently the standard library documentation (i.e. compiler can do whatever it wishes to)

The intent we'd like to convey here is instead that this library is unable to describe what exactly would happen when a `Symbol` with an invalid type is used. But either way the compiler does not receive a permission to do whatever it wishes to.

For that reason I've avoided including the “behaviour” in this documentation. If the old sentence reads poorly, I would rather we delete it entirely. I don't feel like it adds much value.

Reply:
Lets remove it then. Maybe sometime down the line I'll come up with a clarification that's both easy to understand and conveys the intent accurately.